cmd/launch.go

```diff
@@ -273,11 +273,12 @@ func launchNonInteractive(h *ec2helper.EC2Helper) {
 }
 
 // Launch On-Demand or Spot instance based on capacity type
-func LaunchCapacityInstance(h *ec2helper.EC2Helper, simpleConfig *config.SimpleInfo, detailedConfig *config.DetailedInfo, confirmation string) (err error) {
+func LaunchCapacityInstance(h *ec2helper.EC2Helper, simpleConfig *config.SimpleInfo, detailedConfig *config.DetailedInfo,
+	confirmation string, templateID *string) (err error) {
```

Comment:
`LaunchTemplateId` is already a property of `config.SimpleInfo`. Why are we passing it as a separate parameter?